app-backend/api-it/src/test/resources/endpoint-scopes.csv

```diff
@@ -202,4 +202,15 @@ Path,Domain:Action,Verb
 /api/users/me/limits,users:readSelf,get
 /api/users/me/roles,users:readSelf,get
 /api/users/search/,users:search,get
-/api/tasks/random,annotationProjects:readTasks,get
\ No newline at end of file
+/api/tasks/random,annotationProjects:readTasks,get
+/api/campaigns/,campaigns:read,get
+/api/campaigns/,campaigns:create,post
+/api/campaigns/{campaignId},campaigns:read,get
+/api/campaigns/{campaignId},campaigns:update,put
+/api/campaigns/{campaignId},campaigns:delete,delete
+/api/campaigns/{campaignId}/actions,campaigns:readPermissions,get
+/api/campaigns/{campaignId}/projects,annotationProjects:read,get

```

Comment:
Should this be `campaigns:read`? I don't think it follows from `annotationProjects:read` that someone should be able to hit this endpoint

Reply:
Oops this didn't show on the page, but discussed offline, and it's fine / if anyone gets confused the test will show them what's going on